Vienna/Sources/Models/Folder.m

```diff
@@ -151,35 +132,68 @@ -(NSImage * _Nullable)image {
     NSImage *folderImage = nil;
     switch (self.type) {
         case VNAFolderTypeSmart:
-            folderImage = Folder._iconArray[MA_SmartFolderIcon];
+            if (@available(macOS 12, *)) {
+                folderImage = [NSImage imageWithSystemSymbolName:@"folder.badge.gearshape"
+                                        accessibilityDescription:nil];
+            } else if (@available(macOS 11, *)) {
+                folderImage = [NSImage imageWithSystemSymbolName:@"folder.badge.gear"
+                                        accessibilityDescription:nil];
```

Comment:
Do we need to treat separately macOS 11 and macOS 12 ?

Reply:
Yes, `folder.image.gearshape` is the new name for what used to be `folder.badge.gear`. The icon is the same.